packages/node_modules/@ciscospark/storage-adapter-local-storage/src/index.js

```diff
@@ -72,7 +72,7 @@ export default class StorageAdapterLocalStorage {
        */
       clear() {
         loggers.get(this).info(`local-storage-store-adapter: clearing localStorage`);
-        return Promise.resolve(localStorage.clear());
+        return Promise.resolve(localStorage.removeItem(`web-client-internal`));
```

Comment:
This can't be hard coded to `web-client-internal` as there's no telling what value might be passed by a consumer. The key to remove is the key that the rest of the adapter uses.